keyboards/dz60/keymaps/jackokring/keymap.c

```diff
@@ -0,0 +1,265 @@
+/* Copyright 2020 Simon Paul Jackson, BEng.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+/*
+make linux bootloader install via: (or use via if it's firmwared) => CHOICE
```

Comment:
You could use `qmk flash` here.  
Also, shouldn't use sudo to build, it can cause issues, down the line.  Especially if you can use udev rules: https://docs.qmk.fm/#/faq_build?id=linux-udev-rules